__tests__/src/actions/activity.js

```diff
@@ -18,9 +18,9 @@ describe('activity action creators', () => {
   const messagesMock = [{ ...messageMock }, { ...messageMock }];
 
   test('messageAdded', () => {
-    expect(messageAdded(messageMock)).toEqual({
+    expect(messageAdded([messageMock])).toEqual({
```

Comment:
Please keep naming and semantics in line, so if method takes an array from now it should be renamed to `messagesAdded`.